department-of-veterans-affairs/caseflow
# Queue | Colocated and Judge generic actions

### Description
Moves Judge and Colocated tasks over to the `available_actions` generic task model

### Acceptance Criteria
- [ ] ActionsDropdown options for Judges and Colocated users begin checkout flows / navigate to modals as expected

```ruby
routes = {
  assignToUser: "/modal/assign_to_user" (#7214),
  sendToTeam: "/modal/sendToTeam",
  sendToAttorney: "/modal/sendToAttorney",
  placeHold: "place_hold",
  evaluateDecision: "omo_request/evaluate",
  selectDispositions: "dispatch_decision/dispositions",
  specialIssues: "dispatch_decision/special_issues",
}
```

| Docket | Role | Action | Next Route |
|--------|------|--------|------------|
| Legacy | Judge | Assign | `assignToUser` |
| | | Review | `evaluateDecision \|\| selectDispositions` |
| AMA | Judge | Assign | `assignToUser` |
| | | Review | `specialIssues` |
| * | Colocated | * | `sendToAttorney \|\| placeHold` |
| Legacy | Colocated | Translation/Schedule Hearing | `sendToTeam` |

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE`
2. Confirm correct dropdown options appear in Case Details view

#### Known Issues
Judge assignment to attorney does not work yet (#7214)
